### ISSUE TITLE
Allow undercover medics to heal civilians and undercover rebels.

### ISSUE THREAD

**Issue by Spoffy** [official-antistasi-community/A3-Antistasi#172]
Gives undercover a mild buff, and it makes sense. Opens up tactical options too, when the undercover rework comes in.

**Comment by wurzel0701** [comment]
What is the current behaviour? The undercover.sqf does not catch this case

**Comment by Godlikehobbit** [comment]
Current behavior is the "You cannot stay Undercover while: A weapon is visible.. etc" prompt is triggered.
By the looks of it in order to bandage/etc using Ace the player must have a weapon equipped and in the case of the player being undercover they equip a non existent weapon to accomplish this

**Comment by Godlikehobbit** [comment]
Having done some testing, I was correct, if the player has no weapon equipped and attempts to bandage someone they are equipped with ACE_FakePrimaryWeapon

**Comment by Godlikehobbit** [comment]
Having done some testing, changing
https://github.com/official-antistasi-community/A3-Antistasi/blob/c9662602ad2cb2a510abf6c80149bf27bab58362/A3-Antistasi/undercover.sqf#L32 And
https://github.com/official-antistasi-community/A3-Antistasi/blob/c9662602ad2cb2a510abf6c80149bf27bab58362/A3-Antistasi/undercover.sqf#L122 To also check for a lack of ACE_FakePrimaryWeapon: 
```sqf
((primaryWeapon _player != "") and (primaryWeapon _player != "ACE_FakePrimaryWeapon"))
```
This allows an undercover medic to bandage themselves and others. I'm unsure of a good way to enforce allowing only other undercover players/civilians to be bandaged.

**Comment by Spoffy** [comment]
@Godlikehobbit - Are you interested in making a PR for this? :)

**Comment by Godlikehobbit** [comment]
I'm happy to make a PR for it, I'm just working on only allowing undercover medics to heal other undercover players

**Comment by wurzel0701** [comment]
Done and merged, last know issue is #381
